### ISSUE TITLE
Hasura action error

### ISSUE THREAD

**Issue by xavier506** [eoscostarica/eos-rate#596]
Hasura action triggered by a rating is returning the following error

```
==== updating user ratings ====
WARNING: Creating a duplicate database object for the same connection.
    at new Database (/opt/application/node_modules/massive/lib/database.js:76:24)
    at module.exports (/opt/application/node_modules/massive/index.js:32:11)
    at updateUserRatings (/opt/application/src/libs/sync-user-rating.js:33:22)
WARNING: Creating a duplicate database object for the same connection.
    at new Database (/opt/application/node_modules/massive/lib/database.js:76:24)
    at module.exports (/opt/application/node_modules/massive/index.js:32:11)
    at updateStat (/opt/application/src/libs/sync-bp-stats.js:36:11)
NaN
```

Link to Hasura Action 

https://github.com/eoscostarica/eos-rate/blob/88031f17edcfceb5473140481b7e02342846bf5d/services/hasura/metadata/actions.yaml#L5

**Comment by leisterfrancisco** [comment]
Error doesn't appear after some test, but some improvements to sync-bps, proxies and rating will be made

**Comment by leisterfrancisco** [comment]
Some bugs or performance improvements will be solved in #609 #610 #611

**Comment by leisterfrancisco** [comment]
A similar error were found at [add-bps-info](https://github.com/eoscostarica/eos-rate/blob/staging/services/hapi/src/libs/add-bps-info.js#L379), the cause was exist a `for loop` that is calling a function which instantiate `massive` so then next `for interation`, already exist that instantiation and a WARNING message comes up
